third_party/README.rst

```diff
@@ -5,15 +5,15 @@ This part of the repository contains extra dependencies required to build DALI,
-| `Google Benchmark <https://github.com/google/benchmark>`_        | `1.8.3 <https://github.com/google/benchmark/releases/tag/v1.8.3>`_                                                        | `Apache License 2.0 <https://github.com/google/benchmark/blob/master/LICENSE>`_                                   |
+| `Google Benchmark <https://github.com/google/benchmark>`_        | `1.8.4 <https://github.com/google/benchmark/releases/tag/v1.8.4>`_                                                        | `Apache License 2.0 <https://github.com/google/benchmark/blob/master/LICENSE>`_                                   |
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
 | `Boost Preprocessor <https://github.com/boostorg/preprocessor>`_ | `1.85.0 <https://github.com/boostorg/preprocessor/releases/tag/boost-1.85.0>`_                                            | `Boost Software License 1.0 <https://github.com/boostorg/boost/blob/master/LICENSE_1_0.txt>`_                     |
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
 | `COCO API <https://github.com/cocodataset/cocoapi>`_             | `Top-of-tree (Feb 20, 2020) <https://github.com/cocodataset/cocoapi/tree/8c9bcc3cf640524c4c20a9c40e89cb6a2f2fa0e9>`_      | `BSD 2-Clause License <https://github.com/cocodataset/cocoapi/blob/master/license.txt>`_                          |
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
-| `CUTLASS <https://github.com/NVIDIA/cutlass>`_                   | `3.4.1 <https://github.com/NVIDIA/cutlass/tree/v3.4.1>`_                                                                  | `BSD 3-Clause License <https://github.com/NVIDIA/cutlass/blob/master/LICENSE.txt>`_                               |
+| `CUTLASS <https://github.com/NVIDIA/cutlass>`_                   | `3.5.0 <https://github.com/NVIDIA/cutlass/tree/v3.5.0>`_                                                                  | `BSD 3-Clause License <https://github.com/NVIDIA/cutlass/blob/master/LICENSE.txt>`_                               |
```

Comment:
Good luck with that. I tried before - it compiles but the tests fail. It was briefly discussed and we've abandoned the update last time. I doubt it would magically start working a month later.

Reply:
Reverted it